MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -116,7 +116,15 @@ + (IMP)swizzleOriginalSelector:(SEL)originalSelector withCustomSelector:(SEL)cus
   IMP customImp = class_getMethodImplementation(self, customSelector);
   IMP originalImp = NULL;
   BOOL methodAdded = NO;
-  
+
+  // Replace implementation in super classes
+  Class baseClass = originalClass.superclass;
+  if( baseClass != [NSObject class]) {
```

Comment:
Right NSObject is not always the root class but it's most probably is, though it doesn't hurt the way it is right now I think. To me it's maybe more an optimization.